test/tests/functional/pbs_multi_sched.py

```diff
@@ -0,0 +1,601 @@
+# a commercial license agreement.
+#
+# Use of Altair’s trademarks, including but not limited to "PBS™",
+# "PBS Professional®", and "PBS Pro™" and Altair’s logos is subject to Altair's
+# trademark licensing policies.
+
+from tests.functional import *
+
+
+@tags('multisched')
+class TestMultipleSchedulers(TestFunctional):
+
+    """
+    Test suite to test different scheduler interfaces
+    """
+
+    def setUp(self):
+        TestFunctional.setUp(self)
+
+        a = {'partition': 'P1,P4',
+             'sched_host': self.server.hostname,
+             'sched_port': '15050'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc1")
+        self.sched_configure('sc1', '15050')
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 1,
+                             'sched_port': 15050}, id="sc1")
+        dir_path = '/var/spool/pbs/sched_dir'
+        if not os.path.exists(dir_path):
+            self.du.mkdir(path=dir_path, sudo=True)
+        a = {'partition': 'P2',
+             'sched_priv': '/var/spool/pbs/sched_dir/sched_priv_sc2',
+             'sched_log': '/var/spool/pbs/sched_dir/sched_logs_sc2',
+             'sched_host': self.server.hostname,
+             'sched_port': '15051'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc2")
+        self.sched_configure('sc2', '15051', '/var/spool/pbs/sched_dir')
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 1,
+                             'sched_port': 15051}, id="sc2")
+        a = {'partition': 'P3',
+             'sched_host': self.server.hostname,
+             'sched_port': '15052'}
+        self.server.manager(MGR_CMD_CREATE, SCHED,
+                            a, id="sc3")
+        self.sched_configure('sc3', '15052')
+        self.server.manager(MGR_CMD_SET, SCHED,
+                            {'scheduling': 1,
+                             'sched_port': 15052}, id="sc3")
+
+        a = {'queue_type': 'execution',
+             'started': 't',
+             'enabled': 't'}
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq1')
+        print self.server.queues
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq2')
+        print self.server.queues
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq3')
+        print self.server.queues
+        self.server.manager(MGR_CMD_CREATE, QUEUE, a, id='wq4')
+        p1 = {'partition': 'P1'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p1, id='wq1')
+        p2 = {'partition': 'P2'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p2, id='wq2')
+        p3 = {'partition': 'P3'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p3, id='wq3')
+        p4 = {'partition': 'P4'}
+        self.server.manager(MGR_CMD_SET, QUEUE, p4, id='wq4')
+        a = {'resources_available.ncpus': 2}
```

Comment:
line: 88-102
Please validate with "expect=True" for all MGR_CMD_CREATE/MGR_CMD_SET commands